include/aca_message_pulsar_consumer.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2029 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PULSAR_CONSUMER_H
+#define PULSAR_CONSUMER_H
+
+#include "pulsar/Client.h"
+#include "pulsar/Consumer.h"
+#include "pulsar/ClientConfiguration.h"
+#include "pulsar/ConsumerConfiguration.h"
+#include "pulsar/Message.h"
+#include "pulsar/Result.h"
+
+using pulsar::ClientConfiguration;
+using pulsar::ConsumerConfiguration;
+using pulsar::Client;
+using pulsar::Consumer;
+using std::string;
+
+namespace messagemanager
+{
+class MessagePulsarConsumer {
```

Comment:
I just checked other ACA class names (sorry I should have checked it earlier). Please go ahead to name this class ACA_MQ_Pular_Consumer to make it consistent.